investor_wallet_platform_base/models/partner.py

```diff
@@ -93,12 +93,18 @@ def default_structure(self):
     annual_report_link = fields.Char(string="Last annual report link")
     area_char_list = fields.Char(compute='_return_area_char_list',
                                  string="activity areas")
+    industry_char_list = fields.Char(compute='_return_industry_char_list')

```

Comment:
Where is it used? It's probably best to make a call to a route returning a json array.  But I guess _fuck it, lets go forward_.

Reply:
nope it's used by the "vitrine" website. They use xpml-rpc to get the info so no reason to have it as a route.